### PR TITLE
Fix single page interactive sizing

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

**Comment by dougmartin** [comment]
NOTE: The width/height of the predictions is also saved, however there is no width/height in the current prediction json list from S3 so that code is a no-op for now (the React component only posts changed values, since the width/height never change from undefined when switching predictions they are not included in the posted data).

File: app/helpers/interactive_run_helper.rb [hunk 1]
```diff
@@ -56,15 +56,17 @@ def interactive_iframe_tag(interactive,run=nil,iframe_src=true)
       url = interactive_run.learner_url
     end
     url = interactive.url if url.blank?
-    
+
     data = {
       :aspect_ratio => interactive.aspect_ratio,
       :id => interactive.id,
-      :iframe_mouseover => "false"
+      :iframe_mouseover => "false",
```
Comment (scytacki) [inline app/helpers/interactive_run_helper.rb]: this looks like a typo

Comment (dougmartin) [inline app/helpers/interactive_run_helper.rb]: During development I had some other data in there so I added the comma.  I'll remove it.

**Comment by dougmartin** [comment]
@scytacki I've removed the stray comma.

**Comment by scytacki** [comment]
:+1:

**Pull request merged**